it/missione.md

```diff
@@ -16,15 +16,35 @@ Il “sistema operativo” del Paese: una serie di componenti fondamentali sui q
 ## Missione
 
 Rendere i servizi pubblici per i cittadini e aziende accessibili nel modo più semplice possibile, innanzitutto tramite dispositivi mobili (approccio “mobile first”), con architetture sicure, scalabili, altamente affidabili e basate su interfacce applicative (API) chiaramente definite;  supportare le pubbliche amministrazioni centrali e locali nel prendere decisioni migliori e il più possibile basate sui dati, grazie all'adozione delle più moderne metodologie di analisi e sintesi dei dati su larga scala, quali Big Data e Machine Learning.
```

Comment:
per andare a capo serve una riga vuota dopo questo paragrafo (a meno che non vuoi che `In particolare vogliamo...` sia insieme a questo paragrafo

Reply:
Ero indeciso tra le due  ma ho valutato fosse la soluzione migliore.